### ISSUE TITLE
harborUI not accessible using a screen reader

### ISSUE THREAD

**Issue by raikesd** [vmware/harbor#3403]
After installing harbor when I access it using the JAWS screenreader (I am totally blind), I can login, but once logged in when I click the admin button and then select change password, the "current password", "new password", and "verify password" fields are not accessible. I had to have a sighted person help with resetting the password.

The same is true when creating a new project, none of the fields are accessible.

My understanding is that harbor is written using angularJS and if the ngAria model is included it should take care of much of the accessibility problems.

**Comment by steven-zou** [comment]
@raikesd 

  Thanks for raising issue. We'll do some investigation and have discussion with Clarity team which develop the UI library harbor using about the accessible issue.

**Comment by stonezdj** [comment]
To solve the accessibility issue, need to do:

1.Add tableindex="0" for each input element and disable background element in modal dialog.
2.Each pair of label and input should tagged in same div element in the form

**Comment by raikesd** [comment]
Thank you for taking this so seriously.  I have been an accessibility
specialist for the last 17 years, and I always like to let people know
if their applications are not accessible especially if it is an
application I wishto use myself.

Don

**Comment by steven-zou** [comment]
@ninjadq 

  Let's find a proper time to add accessible features into our platform.
